java-manta-client-unshaded/src/main/java/com/joyent/manta/config/BaseChainedConfigContext.java

```diff
@@ -949,13 +955,15 @@ public int hashCode() {
         return Objects.hash(mantaURL, account, mantaKeyId, mantaKeyPath,
                 timeout, retries, maxConnections, privateKeyContent, password,
                 httpBufferSize, httpsProtocols, httpsCipherSuites, noAuth,
-                disableNativeSignatures, tcpSocketTimeout, connectionRequestTimeout, expectContinueTimeout,
+                disableNativeSignatures,
```

Comment:
Why did these lines change?

Reply:
I thought of grouping ```timeouts``` together and ```disableNativeSignatures``` separately to make the code more readable.